deptofdefense/move.mil
# Entitlements copy updates

## Checklist

I have…


## Summary of Changes

This pull request closes several Entitlements-related copy update issues:

- fixes #103 
- fixes #104 
- fixes #105 

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout entitlements-copy-updates`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails rake`,
1. run `bin/rails brakeman`,
1. run `bin/rails rubocop`,
1. run `bin/rails server`,
1. load up http://localhost:3000/entitlements in your Web browser of choice and observe that content and imagery has been updated.
